package.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "ra-auth-cognito-monorepo",
+    "private": true,
+    "author": "Marmelab",
+    "license": "MIT",
+    "devDependencies": {
+        "@babel/preset-env": "^7.15.0",
+        "@babel/preset-react": "^7.14.5",
+        "@babel/preset-typescript": "^7.15.0",
+        "@types/jest": "^29.1.0",
+        "@typescript-eslint/eslint-plugin": "^4.28.5",
+        "@typescript-eslint/parser": "^4.28.5",
+        "babel-jest": "^27.1.0",
+        "eslint": "^7.7.0",
+        "eslint-config-prettier": "^6.11.0",
+        "eslint-config-react-app": "^5.2.1",
+        "eslint-plugin-cypress": "^2.11.1",
+        "eslint-plugin-flowtype": "^5.2.0",
+        "eslint-plugin-import": "^2.22.0",
+        "eslint-plugin-jsx-a11y": "^6.3.1",
+        "eslint-plugin-prettier": "^3.1.4",
+        "eslint-plugin-react": "^7.20.6",
+        "eslint-plugin-react-hooks": "^4.1.0",
+        "jest": "^29.1.1",
+        "jest-environment-jsdom": "^29.1.1",
+        "prettier": "~2.1.1",
+        "raf": "^3.4.1",
+        "ts-jest": "^29.0.3",
+        "typescript": "^4.8.4"
+    },
+    "scripts": {
+        "build": "yarn workspaces run build",
+        "build-demo": "cd packages/demo && yarn build",
+        "run-demo": "cd packages/demo && yarn dev",
+        "test-unit": "cd packages/ra-autho0 && yarn test-unit",
```

```suggestion
        "test-unit": "cd packages/ra-auth-cognito && yarn test-unit",
```
I we needed proof there are no tests, there it is! :rofl: